e2e-tests/cypress/tests/support/ui/search.ts

```diff
@@ -19,4 +30,15 @@ Cypress.Commands.add('uiJumpToSearchResult', (postId) => {
 
     // * Verify that the permalinked post is highlighted in the center channel
     cy.get(`#post_${postId}.post--highlight`).should('be.visible');
-});
+}
+
+// eslint-disable-next-line @typescript-eslint/no-use-before-define
+Cypress.Commands.add('uiJumpToSearchResult', uiJumpToSearchResult);
```

Comment:
Let's remove this disable/flag and the other instance in this file by updating the global declaration below with:
```javascript
declare global {
    // eslint-disable-next-line @typescript-eslint/no-namespace
    namespace Cypress {
        interface Chainable {
            uiSearchPosts: typeof uiSearchPosts;
            uiJumpToSearchResult: typeof uiJumpToSearchResult;
        }
    }
}

export {};
```

```suggestion
Cypress.Commands.add('uiJumpToSearchResult', uiJumpToSearchResult);
```

Reply:
Thanks for your suggestions.

Done!